docs/release/release-tasks.md

```diff
@@ -337,16 +337,25 @@ The goal of this task to make the book for the current release available under e
 
 #### Generate weekly PR updates to post in Slack
 The goal of this task is to keep the CAPI community updated on recent PRs that have been merged. This is done by using the weekly update tool in `hack/tools/release/weekly/main.go`. Here is how to use it:
-1. Checkout the latest commit on the release branch, e.g. `release-1.6`, or the main branch if the release branch doesn't yet exist (e.g. beta release).
-2. Build the release weekly update tools binary.
+1. Build the release weekly update tools binary.
    ```bash
    make release-weekly-update-tool
    ```
-3. Generate the weekly update with the following command:
+2. Generate the weekly update with the following command:
    ```bash
-   ./bin/weekly --from YYYY-MM-DD --to YYYY-MM-DD --milestone v1.x
+   ./bin/weekly --since YYYY-MM-DD --until YYYY-MM-DD --branch release-1.x
```

Comment:
I think you can add a note here to mention setting the `GITHUB_TOKEN` environment variable can reduce rate limiting issues.

Reply:
Done, thanks for the suggestion :)